CMakeLists.txt

```diff
@@ -682,6 +682,7 @@ enable_clang_tidy(
         -cppcoreguidelines-avoid-capture-default-when-capturing-this
         -cppcoreguidelines-rvalue-reference-param-not-moved
         -readability-avoid-unconditional-preprocessor-if
+        -bugprone-lambda-function-name
```

Comment:
I guess it's a bit unrelated to initial PR, but probably it's ok.

Reply:
@CAHEK7 it is related here: [Clang Tidy Log](http://micimaster.amd.com/blue/rest/organizations/jenkins/pipelines/MLLibs/pipelines/MIOpen/branches/PR-2312/runs/2/nodes/88/log/?start=0)

```
[2023-09-13T06:02:15.863Z] /home/jenkins/workspace/MLLibs_MIOpen_PR-2312/src/solver/gemm_wrw.cpp:212:17: error: inside a lambda, '__FUNCTION__' expands to the name of the function call operator; consider capturing the name of the enclosing function explicitly [bugprone-lambda-function-name,-warnings-as-errors]
[2023-09-13T06:02:15.863Z]                 MIOPEN_LOG_FUNCTION("groupconv, 1x1");
[2023-09-13T06:02:15.863Z]                 ^
[2023-09-13T06:02:15.863Z] /home/jenkins/workspace/MLLibs_MIOpen_PR-2312/src/include/miopen/logger.hpp:326:49: note: expanded from macro 'MIOPEN_LOG_FUNCTION'
[2023-09-13T06:02:15.863Z]             miopen_log_func_ss << "s_api = " << __FUNCTION__ << " | ";                  \
[2023-09-13T06:02:15.863Z]                                                 ^
```